src/helpers/layout.js

```diff
@@ -396,13 +397,22 @@ class LayoutHelper {
         if(this.ctx.zoomedTrack != ''){
             let prevZoomIconEle = this.ctx.querySelector('.pvZoomIcon_'+this.ctx.zoomedTrack);
             prevZoomIconEle.classList.remove('active');
+            const icon=prevZoomIconEle.children[0];
+            const text=prevZoomIconEle.children[1];
+            icon.classList.remove("icon-search-minus");
+            icon.classList.add("icon-search-plus");
+            text.innerText="Zoom in";
```

Comment:
This pattern is repeated 4 times, right? (with different element/classes/text). We can abstract it to a function.